drivers/media/platform/raspberrypi/rp1_cfe/cfe.c

```diff
@@ -1166,6 +1166,9 @@ static int cfe_start_streaming(struct vb2_queue *vq, unsigned int count)
 		goto err_disable_cfe;
 	}
 
+	cfe->csi2.dphy.noncontinuous_clock =
+		cfe->csi2.bus_flags & V4L2_MBUS_CSI2_NONCONTINUOUS_CLOCK;
+
```

Comment:
Note that clocking mode can also be signaled via the `get_mbus_config` call that is handled in `cfe_start_streaming`.
That's used if you have something like a CSI2 mux with different inputs using different clocking modes (or number of lanes, which is already handled).

Reply:
Fixed!

Reply:
Should be:

```c
	cfe->csi2.dphy.noncontinuous_clock = mbus_config.bus.mipi_csi2.flags &
					     V4L2_MBUS_CSI2_NONCONTINUOUS_CLOCK;
```